src/main/java/com/google/cloud/tools/appengine/cloudsdk/process/LegacyProcessHandler.java

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.cloudsdk.process;
+
+import com.google.cloud.tools.appengine.api.AppEngineException;
+import com.google.cloud.tools.appengine.cloudsdk.internal.process.ProcessRunnerException;
+import com.google.cloud.tools.appengine.cloudsdk.internal.process.WaitingProcessOutputLineListener;
+import com.google.common.base.Charsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/** Process handler that mimics the previous behavior of ProcessRunner. */
+public class LegacyProcessHandler implements ProcessHandler {
```

Comment:
This is just a version of https://github.com/GoogleCloudPlatform/appengine-plugins-core/blob/master/src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/process/ProcessRunner.java